src/AspNet.Security.OpenId.Steam/SteamAuthenticationHandler.cs

```diff
@@ -94,5 +94,15 @@ public class SteamAuthenticationHandler : OpenIdAuthenticationHandler<SteamAuthe
             // with the notification to allow replacing the ticket.
             return context.Ticket;
         }
+
+        private void StoreProfileItemInClaim(ClaimsIdentity identity, JObject payload, string ClaimType, string param) {
+            var profile = payload.Value<JObject>(SteamAuthenticationConstants.Parameters.Response)
```

Comment:
Have you considered moving that outside the method so we don't parse the profile multiple times?